fiware-region-sanity-tests/commons/swift_operations.py

```diff
@@ -85,13 +85,47 @@ def list_containers(self, name_prefix=None):
 
         return container_list
 
-    def create_container(self, containerName):
+    def create_container(self, container_name):
         """
         Creates a new Container
         :param container_name: Name of the container
         :return: None if container was created and the error message if something failed.
         """
-        response = self.client.put_container(containerName)
+        response = self.client.put_container(container_name)
+        return response
+
+    def create_text_object(self, container_name, local_file, object_name):
+        """
+        Creates a new text object into a container
+        :param container_name: Name of the container
+        :return: None if container was created and the error message if something failed.
+        """
+        with open(local_file, 'r') as test_file:
+            self.client.put_object(container_name, object_name,
+                                        contents=test_file.read(),
+                                        content_type=TEXT_PLAIN)
+
+    def get_text_object(self, container_name, object_name, download_path):
+        """
+        Downloads a text object from a container
+        :param container_name: Name of the container
+        :param object_name: Name of the container
+        :param download_path: Path to download folder
+        :return: None if container was created and the error message if something failed.
+        """
+        obj = self.client.get_object(container_name, object_name)
```

Comment:
I do not follow the code, but it is not possible that self.client.get_object returns an error or null obj?